core/src/main/java/org/mobilitydata/gtfsvalidator/validator/ValidationContext.java

```diff
@@ -48,15 +48,26 @@ public static Builder builder() {
    *   <li>unit tests may need to override the current time.
    * </ul>
    *
-   * @return The time when validation started as @code{ZonedDateTime}
+   * @return The time when validation started as @{code CurrentDateTime}
```

Comment:
Once again: {@code CurrentDateTime} :)

Reply:
Shame on me 🙈. Done. Thanks.

Reply:
I also often make a typo in that {@code}. I wonder if there are automatic checks for Javadoc syntax.